lib/ui/platform_dispatcher.dart

```diff
@@ -2605,3 +2662,52 @@ class SemanticsActionEvent {
     );
   }
 }
+
+/// Signature for [PlatformDispatcher.onPlatformFocusChange].
+typedef ViewFocusChangeCallback = void Function(ViewFocusEvent);
+
+/// An event for the engine to communicate view focus changes back to the app.
```

Comment:
Leave a breadcrumb in this doc back to the [requestViewFocusChange] API.

Reply:
Done